index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="container">
+        <div class="logoholder">
+            <b class="mylogo">My Logo</b>
+        </div>
+        <div class="menuholder">
+            <img src="menuicon.png" alt="test">
+
+        </div>
+
+       
+
+        
+    </div>
+    <div class="container2">
+        <div class="headerholder"> 
+            <h2 class="head">I’m Mariane<br>Glad to see<br> you!</h2>
+        </div>
+        <p class="headtext">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+        <div class="supporttext">
+            <b class="suptext">
+                LETS CONNECT
+            </b>
+        </div>
+        
+        <div class="social">
+            <img src="Vector.svg" alt="">
+            <img src="Linkedinicon.svg" alt="">
+            <img src="Vector.png" alt="">
+            <img src="Vector2.svg" alt="">
+            <img src="Vector3.svg" alt="">
+        </div>
+
+    </div>
```

Comment:
**REQUIRED:** The second part(headline) should be inside the `section` tag(you have used `div`), as I mentioned above try to implement semantic HTML.